problem/views.py

```diff
@@ -70,6 +71,9 @@ def detail(request, pid):
 
 @login_required
 def edit(request, pid=None):
+    SPECIAL_PATH = config_info.get_config('path', 'special_judge_path')
```

Comment:
Can you move that out of the function?
Otherwise, it will query config file each time you edit.